tests/integration/telemetry/stackdriver/testdata/custom_bootstrap.yaml.tmpl

```diff
@@ -30,7 +29,7 @@ data:
             "stackdriver_exporter_enabled": true,
             "stackdriver_grpc_service": {
               "google_grpc": {
-                "target_uri": "{{ .StackdriverAddress }}",
+                "target_uri": "cloudtrace.googleapis.com",
```

Comment:
Why are we testing with real sd backend?

Reply:
I'm testing the audit function with real stackdriver and tried to debug if it works for request-header field matching of authz policy. But the current code does not work for request-header field matching, and I will need to enable rbac envoy filter for the integration test. Do you have any recommendation on how to enable rbac envoy filter debugging for integ test?

Reply:
I think you can psss `--istio.test.kube.helm.values="global.proxy.componentLogLevel=rbac:debug"` to enable the component logging when running the test, it really doesn't make sense that the request-header matching not working while the path matching is working, we don't do anything special to request-header matching.